src/legacy/compiler/ts-compiler.spec.ts

```
@@ -405,6 +405,48 @@ describe('TsCompiler', () => {
         expect(compiler._fileContentCache.has(emptyFile)).toBe(true)
       })
 
+      test.each([
+        { version: '6.0.0', expectedResolution: ts.ModuleResolutionKind.Bundler },
+        { version: '6.1.2', expectedResolution: ts.ModuleResolutionKind.Bundler },
+        { version: '6.0.0-dev.20251015', expectedResolution: ts.ModuleResolutionKind.Bundler },
+        { version: '5.9.3', expectedResolution: ts.ModuleResolutionKind.Node10 },
+        { version: '4.7.4', expectedResolution: ts.ModuleResolutionKind.Node10 },
+      ])(
+        'should default moduleResolution to %p for TypeScript $version',
+        ({ version, expectedResolution }) => {
```

ESLint

[error] Replace `⏎········'should·default·moduleResolution·to·%p·for·TypeScript·$version',⏎·······` with `'should·default·moduleResolution·to·%p·for·TypeScript·$version',`

(prettier/prettier)